docs/changelogs/unreleased.md

```diff
@@ -0,0 +1,88 @@
+This is the second release candidate for the upcoming Cozystack v0.31.0 release.
+
+Cozystack 0.31.0 further advances GPU support, monitoring, and all-around convenience features.
+
+## New Features and Changes
```

Comment:
Let's take an example from [kubernetes' changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.33.md). It would be nice to have a header somewhere stating that these are changes since v0.30.0 or v0.31.0-rc.1 or whatever. As a maintainer, I would actually prefer to have the `UNRELEASED.md` document list changes since v0.31.0-rc.1. This would make it easier for me to compile the next release by just picking the specific PRs that are only present in `main` and not in the latest release candidate.

Reply:
I have the following workflow in my mind:

* When we publish a stable release and stop merging things to `main`, make this file `unreleased.md` there
* All the way to a new stable version, add lines in the file.
* On a release candidate, take what we have and use it as release notes. Then, add more lines.
* On a stable release, update the file and rename it to `v0.31.0.md`, for example